test/integration/server/utils/snapshot-client.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import * as polarisProperties from "../resources/polaris-properties.json";
+
+export const url = `http://localhost:${polarisProperties.port}/snapshot`;
```

Comment:
If you are just using the version, import it directly, like this:
`import { version } from "../resources/polaris-properties.json";`